walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -29,9 +29,18 @@
    * Returns the open windows for which {@code request} can be scheduled in.
    */
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    Collection<String> requiredAttendees = request.getAttendees();
-    Collection<TimeRange> availableTimes = assembleTime(events, requiredAttendees, request.getDuration());
-    return adjustAvailableTimes;
+    Collection<String> requiredAttendees = new ArrayList<>(request.getAttendees());
+    Collection<String> optionalAttendees = request.getOptionalAttendees();
+    Collection<String> allAttendees = new ArrayList<>(requiredAttendees);
+    allAttendees.addAll(optionalAttendees);
```

Comment:
We could probably reuse the first list
```suggestion
    Collection<String> attendees = new ArrayList<>(request.getAttendees());
    Collection<String> optionalAttendees = request.getOptionalAttendees();
    attendees.addAll(optionalAttendees);
```
WDYT?

Reply:
I need two lists one with the required attendees only and one with both the optional attendees and the required attendees. The suggestion has one list with optional and required attendees and an optional attendee list.